src/CFamily/Analysis/CLangAnalyzer.cs

```diff
@@ -122,7 +126,7 @@ public bool IsAnalysisSupported(IEnumerable<AnalysisLanguage> languages)
             // For notes on VS threading, see https://github.com/microsoft/vs-threading/blob/master/doc/cookbook_vs.md
```

Comment:
Minor: I would probably delete this comment: we're now using our own abstraction layer, so the cookbook isn't directly relevant.